From 0b40944b0a57393ccedb19b18f5c8307cfd42ffe Mon Sep 17 00:00:00 2001
From: Kemeng Shi <shikemeng@huawei.com>
Date: Tue, 19 Oct 2021 10:43:09 +0800
Subject: [PATCH 02/33] etmem: split example_conf.yaml

User can't use example_conf.yaml directly as it contains diffrent
engine configs. Split it into three single engine config and user
can use it directly

Signed-off-by: Kemeng Shi <shikemeng@huawei.com>
---
 etmem/conf/cslide_conf.yaml     | 24 ++++++++++++++++++
 etmem/conf/example_conf.yaml    | 56 -----------------------------------------
 etmem/conf/slide_conf.yaml      | 18 +++++++++++++
 etmem/conf/thirdparty_conf.yaml | 21 ++++++++++++++++
 4 files changed, 63 insertions(+), 56 deletions(-)
 create mode 100644 etmem/conf/cslide_conf.yaml
 delete mode 100644 etmem/conf/example_conf.yaml
 create mode 100644 etmem/conf/slide_conf.yaml
 create mode 100644 etmem/conf/thirdparty_conf.yaml

diff --git a/etmem/conf/cslide_conf.yaml b/etmem/conf/cslide_conf.yaml
new file mode 100644
index 0000000..6b6ecc3
--- /dev/null
+++ b/etmem/conf/cslide_conf.yaml
@@ -0,0 +1,24 @@
+[project]
+name=test
+loop=1
+interval=1
+sleep=1
+
+#cslide
+[engine]
+name=cslide
+project=test
+node_pair=0,2;1,3
+hot_threshold=1
+node_mig_quota=1024
+node_hot_reserve=1024
+
+[task]
+project=test
+engine=cslide
+name=background_cslide
+type=pid
+value=23456
+vm_flags=ht
+anon_only=no
+ign_host=no
diff --git a/etmem/conf/example_conf.yaml b/etmem/conf/example_conf.yaml
deleted file mode 100644
index 61ee68c..0000000
--- a/etmem/conf/example_conf.yaml
+++ /dev/null
@@ -1,56 +0,0 @@
-[project]
-name=test
-loop=1
-interval=1
-sleep=1
-
-#slide
-[engine]
-name=slide
-project=test
-
-[task]
-project=test
-engine=slide
-name=background_slide
-type=name
-value=mysql
-T=1
-max_threads=1
-
-#cslide
-[engine]
-name=cslide
-project=test
-node_pair=0,2;1,3
-hot_threshold=1
-node_mig_quota=1024
-node_hot_reserve=1024
-
-[task]
-project=test
-engine=cslide
-name=background_cslide
-type=pid
-value=23456
-vm_flags=ht
-anon_only=no
-ign_host=no
-
-#thirdparty
-[engine]
-name=thirdparty
-project=test
-eng_name=my_engine
-libname=/usr/lib/etmem_fetch/my_engine.so
-ops_name=my_engine_ops
-engine_private_key=engine_private_value
-
-[task]
-project=test
-engine=my_engine
-name=backgroud_third
-type=pid
-value=12345
-task_private_key=task_private_value
-
diff --git a/etmem/conf/slide_conf.yaml b/etmem/conf/slide_conf.yaml
new file mode 100644
index 0000000..b99ab50
--- /dev/null
+++ b/etmem/conf/slide_conf.yaml
@@ -0,0 +1,18 @@
+[project]
+name=test
+loop=1
+interval=1
+sleep=1
+
+[engine]
+name=slide
+project=test
+
+[task]
+project=test
+engine=slide
+name=background_slide
+type=name
+value=mysql
+T=1
+max_threads=1
diff --git a/etmem/conf/thirdparty_conf.yaml b/etmem/conf/thirdparty_conf.yaml
new file mode 100644
index 0000000..1e1e9ac
--- /dev/null
+++ b/etmem/conf/thirdparty_conf.yaml
@@ -0,0 +1,21 @@
+[project]
+name=test
+loop=1
+interval=1
+sleep=1
+
+[engine]
+name=thirdparty
+project=test
+eng_name=my_engine
+libname=/usr/lib/etmem_fetch/my_engine.so
+ops_name=my_engine_ops
+engine_private_key=engine_private_value
+
+[task]
+project=test
+engine=my_engine
+name=backgroud_third
+type=pid
+value=12345
+task_private_key=task_private_value
-- 
1.8.3.1

